### DISCUSSION TITLE
Note keeping feature

### DISCUSSION THREAD

**Discussion by dooley74** [Ylianst/MeshCentral#4106] [Ideas]
I was thinking it would be nice if there is a place on the My Devices tab where we can store snippets of text, for easy access to often used powershell or cmd lines of code **AND** have it seen by all admin/tech accounts, or be able to assign who can see it.
I realise that there is a notes feature already but the other users/techs/admins in the mesh server cant see my notes I have stashed there.

**Comment by Ylianst** [comment]
Device notes are stored per-device, so, anyone that has access to the device should be able to see your notes. I do agree that this may be an undervalued feature and should be improved. I see #4091 was just opened related to this issue.
